src/protocols/chainsync.rs

```diff
@@ -238,76 +238,82 @@ impl Protocol for ChainSyncProtocol {
         //msgIntersectNotFound   = [6, tip]
         //chainSyncMsgDone       = [7]
 
-        let cbor_value: Value = de::from_slice(&data[..]).unwrap();
```

Comment:
What is the functional change here? What I see is an additional level of indentation resulting from the improved error handling which we can further clean up later. It unfortunately hides the real changes at least in GitHub's view.

Reply:
Simply just handling the error when unwrap fails. One user reported a crash here.  The real change for the whole PR is just putting in the correct final byron block for the guild network.